### ISSUE TITLE
LSTM attention mechanism 

### ISSUE THREAD

**Issue by ForeverZyh** [robinjia/certified-word-sub#2]
Hi, in the paper you define the IBP for softmax which is used in the LSTM attention mechanism. However, I only found the 'mean' pooling for training LSTM.
Maybe I missed it in your paper, but my question is: Did you use the mean pooling for LSTM in your evaluation?

**Comment by robinjia** [comment]
Our main experiments with LSTM use mean pooling. There is also an experiment in Appendix A.7 that uses the final state of the LSTM, instead of mean pooling. We did not run experiments that combine the attention mechanism with an LSTM, although this should be doable with the code.
